<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>836</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Categories(Comma separated)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_categories">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Tags (Comma Separated)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_tags"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Favorite?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_fav">
          <property name="statusTip">
           <string notr="true"/>
          </property>
          <property name="whatsThis">
           <string notr="true"/>
          </property>
          <property name="accessibleName">
           <string notr="true"/>
          </property>
          <property name="accessibleDescription">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_HIP">
   <property name="text">
    <string>$HIP</string>
   </property>
  </action>
  <action name="action_HOME">
   <property name="text">
    <string>$HOME</string>
   </property>
  </action>
  <action name="actionCreate_Thumbs_for_Directory">
   <property name="text">
    <string>Recursively Create Thumbs for this Folder (slow)</string>
   </property>
  </action>
  <action name="actionRefresh_Current_Dir">
   <property name="text">
    <string>Recreate Thumbs for this Folder</string>
   </property>
  </action>
  <action name="actionClear_Thumb_Database">
   <property name="text">
    <string>Clear All Materials</string>
   </property>
  </action>
  <action name="action50">
   <property name="text">
    <string>50</string>
   </property>
  </action>
  <action name="actionAddFavourite">
   <property name="text">
    <string>Add Current Folder to Favourites</string>
   </property>
  </action>
  <action name="actionStartup">
   <property name="text">
    <string>Set Current Library as Default</string>
   </property>
  </action>
  <action name="actionRemFavourite">
   <property name="text">
    <string>Remove Current Folder from Favourites</string>
   </property>
  </action>
  <action name="action_updateAll">
   <property name="text">
    <string>Update All Materials</string>
   </property>
  </action>
  <action name="action_setDefault">
   <property name="text">
    <string>Set Current Library as Default</string>
   </property>
  </action>
  <action name="action_openLibrary">
   <property name="text">
    <string>Open Library</string>
   </property>
  </action>
  <action name="action_prefs">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
